sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -524,7 +530,7 @@ class PlanParserSuite extends PlanTest {
       Hint("STREAMTABLE", Seq("a", "b", "c"), table("t").select(star())))
 
     comparePlans(
-      parsePlan("SELECT /*+ INDEX(t emp_job_ix) */ * FROM t"),
+      parsePlan("SELECT /*+ INDEX(t, emp_job_ix) */ * FROM t"),
```

Comment:
Hi, @viirya and @cloud-fan @hvanhovell @rxin @gatorsmile .

Just for explanation, it was originally designed to support other syntax like the followings. Except query block parts (which we don't support), this case means `tableSpec` and `indexSpec`.

https://docs.oracle.com/cd/B12037_01/server.101/b10752/hintsref.htm#21629

At that time, there was a request to provide more general syntax to prevent future changes on `SqlBase.g4` layer.

Anyway, I have no objection on the current approach since I also read the comments between @rxin and @viirya on the previous @rxin 's commit.